Jenkinsfile

```diff
@@ -110,9 +111,11 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
         }
 
         stage('Fetch') {
-            if (s3_builddir) {
+            if (s3_stream_dir) {
                 utils.shwrap("""
-                coreos-assembler buildprep s3://${s3_builddir}
+                coreos-assembler buildprep s3://${s3_stream_dir}/builds
+                # also fetch releases.json to get the latest build, but don't error if it doesn't exist
+                aws s3 cp s3://${s3_stream_dir}/releases.json tmp/releases.json || true
```

Comment:
Stuff like this is dangerous, really want to only ignore `ENOENT`.

Reply:
Agreed (`buildprep` does this); I just cut a corner here since there's no easy way to do this from the CLI. "Dangerous" might be a bit too strong a word though; regardless of whether this fails or not, we will always need to sanity check the parent as part of our release process (manually for now, but encoded into our release tools too in the future).

I guess since we're in the cosa container, we *could* `sh "python3 -c ' ... '"` so we can use the boto3 API directly...

Reply:
Fixed! :arrow_down: